### PR TITLE
Improve the text on the home page

### PR DESCRIPTION
**Pull request by lalitpatel**
### Background and Goal
The current homepage text is not much helpful for the a user who is visiting the website for the first time. The first thing they notice on the home page is the "Found any issue" notice which is probably not something they are looking for. 

This PR improves the text on the home page with the following objectives:
* A new users should be able to get a basic idea about the purpose of the tool.
* Present couple of CTA's to the user.
* Move the notice to the bottom of the main content.

#### Todo
- [x] Add Chinese translations for the new strings
- [x] Add Spanish translations for the new strings
- [x] Add German translations for the new strings

### CODE REVIEW

**Comment by HSZemi** [comment]
https://github.com/SiegeEngineers/aoe2cm2/tree/feature/home-page-de-translation

File: src/components/PresetEditor/HelpText.tsx [hunk 1]
```diff
@@ -19,6 +19,9 @@ export const HelpBasics = withTranslation()(() =>
                 case of timeout a random civilization is picked for the captain or no civilization
                 gets banned.</Trans>
         </p>
+        <p>
+            <Trans i18nKey='instructions.basics.3'>For each turn captains have <b>30 seconds</b>. In case of a timeout, a random civilisation is selected for the captain.</Trans>
+        </p>
```
Comment (HSZemi) [inline src/components/PresetEditor/HelpText.tsx:24]: I think this is worth adding to the start page as well

Comment (lalitpatel) [reply src/components/PresetEditor/HelpText.tsx]: Added.

**Comment by HSZemi** [comment]
Thank you :yellow_heart:

**Comment by HSZemi** [comment]
Thanks to everyone involved :yellow_heart:

**Pull request merged**